### ISSUE TITLE
Mitype complains about missing _curses module on Windows

### ISSUE THREAD

**Issue by Vinzent03** [Mithil467/mitype#65]
### Subject of the issue
When I start mitype with `mitype` in cmd on **windows** I get the error `ModuleNotFoundError: No module named '_curses'` In some research I found out, that `curses` is not really supported on windows. To fix this problem I had to run`pip install windows-curses`. I think that should be documentated.

### Your environment
* windows
* mitype: 0.2.0
* python: 3.8.5
* terminal: windows terminal

### Steps to reproduce
after a fresh installation of `mitype` run `mitype` on **windows**
### Expected behaviour
It should be able to start.
### Actual behaviour
```
PS C:\Users\user> mitype
Traceback (most recent call last):
  File "c:\users\user\appdata\local\programs\python\python38-32\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\user\appdata\local\programs\python\python38-32\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\user\AppData\Local\Programs\Python\Python38-32\Scripts\mitype.exe\__main__.py", line 4, in <module>
  File "c:\users\user\appdata\local\programs\python\python38-32\lib\site-packages\mitype\app.py", line 3, in <module>
    import curses
  File "c:\users\user\appdata\local\programs\python\python38-32\lib\curses\__init__.py", line 13, in <module>
    from _curses import *
ModuleNotFoundError: No module named '_curses'
```

### Your proposed fix (if any)
run `pip install windows-curses`

**Comment by Mithil467** [comment]
Hey @Vinzent03 :wave:

This was documented earlier, but PR #45  was merged to make it such that this dependency is automatically resolved on windows. So for upcoming releases this is supposed to work out of the box. Although, I see that currently there is no way for users to know that they need `windows_curses` for mitype v0.2.0 and lesser.

v0.2.1 will be out very soon, and users won't face this issue any more.

I have added this information in [BUILD.md](https://github.com/Mithil467/mitype/blob/master/BUILD.md) so that windows users do not face any difficulty while running mitype directly from source.

Thank you very much for pointing it out, I really appreciate the trouble taken to find the cause and filing an issue. 🙌🏼
